### DISCUSSION TITLE
BUG? Include css like External style sheets, but not like Embedded styles.

### DISCUSSION THREAD

**Discussion by peektoseen** [nuxt/nuxt#9406] [Questions (Nuxt 2)] [answered]
Hello everyone!

Hi all! 
I develop web site on nuxt.js and try use bootstrap-vue module.
What I do: 
```
$ mkdir nuxttest
$ cd nuxttest
$ yarn create nuxt-app test
$ cd test/
$ yarn add bootstrap-vue
```

After that - edit /nuxt.config.js, and add **bootstrap-vue/nuxt** in modules section:
```
  modules: [
    "bootstrap-vue/nuxt"
  ],
```  

```
$ yarn build
$ yarn start
```

And open in browser (chrome) source code of page. I see, that bootstrap styles included embedded (like <style>...</style>). 
I want **connect bootstrap styles like file** in `<link rel="stylesheet" ... >` how can i make it?

I try ask on bootsrap-vue repository [issues](https://github.com/bootstrap-vue/bootstrap-vue/issues/6652), but they dirrect me to here. 
![2021-06-08_13-25](https://user-images.githubusercontent.com/2859889/121169815-108b2400-c85d-11eb-86af-5d25a8fb8147.png)

I attach screenshot, where show how  bootsrap css included embedded in nuxt.js , but I need include it from file like `<link href='..`

**Comment by peektoseen** [comment]
Somebody used bootstrap-vue with nuxt? How your styles included into final web page sources in browser? I suggest, it may be related to webpack configuration.

**Comment by danielroe** [comment, marked as answer]
Yes, this is an issue that should be fixed, reported at https://github.com/nuxt/nuxt.js/issues/8569.

**Reply by peektoseen** [reply]
Oh, thank you very much! I will follow this issue.
